lib/auth/init.go

```diff
@@ -71,14 +71,30 @@ type InitConfig struct {
 
 	// Trust is a service that manages users and credentials
 	Trust services.Trust
+
 	// Lock is a distributed or local lock service
 	Lock services.Lock
+
 	// Presence service is a discovery and hearbeat tracker
 	Presence services.Presence
+
 	// Provisioner is a service that keeps track of provisioning tokens
 	Provisioner services.Provisioner
+
 	// Trust is a service that manages users and credentials
 	Identity services.Identity
+
+	// StaticTokens are pre-defined host provisioning tokens supplied via config file for
+	// environments where paranoid security is not needed
+	StaticTokens []StaticToken
+}
+
+// StaticToken describes a pre-defined always-live token, usually stored in a config file.
```

Comment:
I think `StaticToken` should be named `ProvisioningToken` as it's not clear what `Static` means however this token is used for `Provisioning` a new server